Anna_Gamzinova/MongoDB/objects/base_db.py

```diff
@@ -0,0 +1,34 @@
+import pymongo
+
+
+class BaseDB:
+    def __init__(self, database, collection):
+        self.connection = pymongo.MongoClient("mongodb://127.0.0.1:27017/")
+        self.database = self.connection[database]
+        self.collection = self.database[collection]
+
+    def insert_value(self, new_name, new_color, new_occupation):
```

Comment:
it is not a base function. 
What if I want to insert value for other instance which do not have such fields?

Reply:
@bohdandovbsyh removed to cars